src/components/Navbar.tsx

```diff
@@ -0,0 +1,41 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import LogoutButton from "./LogoutButton";
+
+function Navbar() {
+  const { isAuthenticated } = useAuth0();
+
+  return (
+    <nav className="navbar navbar-expand-lg bg-body-tertiary">
+      <div className="container-fluid header">
+        <a className="navbar-brand" href="#">
```

Comment:
Będzie to link? Jeżeli nie, to lepiej zamienić to na button, albo jeśli po prsotu to będzie tekst to na jakis nagłówek.

Reply:
Nie wiem jeszcze, to nadal nie mój kod jak będzie logika działać to będę stalować po swojemu, a to z googla

Reply:
To trochę bez sensu. Żeby auth działał, nie trzeba dodawać wielu różnych elementów i stylów które z założenia będą kiedyś wywalone. Zeby przetestować auth, wystarczy goły html nawet. Wiec dodawanie wielu rzeczy "na chwile" uposledza wlasnie proces code-review przez to ze nei wiem co jest Twoje, a co zaraz zniknie. Brudzi nam tez historię commitow, bo dodajemy nadmiar kodu ktory i tak zniknie. Minimum zmian potrzebnych do wykonania zadania, keep it simple oraz YAGNI, to podstawowe zasady w programowaniu.